README.md

```diff
@@ -49,7 +49,7 @@ app.post('/graphql', graphqlHTTP({
 
 app.get('/graphql', graphqlHTTP({
   schema: MyGraphQLSchema,
-  graphiql: true // set it to false to disable the GraphiQL interface
```

Comment:
I don't think we're worried about that. Look at all the other places in the README where we show `graphiql: true` without any comments. If we're concerned that this setting is non-obvious it should be renamed, and if we're concerned that mis-use of it is dangerous, then we've got the wrong API. But I don't think either of those are the case.